src/components/compoundComponents/Breadcrumbs/breadcrumbs.mdx

```diff
@@ -0,0 +1,117 @@
+import BreadcrumbsExample from './BreadcrumbsExample';
+
+<CH.Scrollycoding>
+
+    # Building a compound component
+
+    <CH.Preview frameless>
+
+        <BreadcrumbsExample />
+
+    </CH.Preview>
+
+    ```tsx BreadcrumbsExample.tsx
+
+        import React from 'react';
+
+        import Breadcrumbs from './Breadcrumbs';
+
+        function App() {
+            // TODO: Replace this:
+            return (
+                <nav aria-label="Breadcrumbs">
+                    <ol>
+                        <li>
+                            <a href="/">Electronics</a>
+                        </li>
+                        <li>
+                            <a href="/cellphones">Cell Phones</a>
+                        </li>
+                        <li>
+                            <a href="/cellphones/cases" aria-current="page">
+                                Cases
+                            </a>
+                        </li>
+                    </ol>
+                </nav>
+            );
+        }
+
+        export default App;
+
+    ```
+
+    E-commerce websites often have “breadcrumbs” to help the user navigate to parent/grand-parent pages. For example:
```

Comment:
Would move this outside of the scroll coding so that the description inside the scroll coding is only about the code. And the explanation happens before that